website/src/modules/docs/pages/page.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Loadable from 'react-loadable';
+import { RouteComponentProps } from 'react-router';
+
+import DocsDashboard from '../components/DocsDashboard';
+import DocsDashboardContent from '../components/DocsDashboardContent';
+import Loading from '../../core/components/Loading';
+
+interface RouteParams {
+  page: string;
+}
+
+const DocsPage: React.SFC<RouteComponentProps<RouteParams>> = ({ match }) => {
+  console.log(match.params.page);
```

Comment:
is it intended?

Reply:
Absolutely not. I forgot to remove that. 😅